src/widgets/context-menu.ts

```diff
@@ -19,6 +19,6 @@ export class ContextMenu extends Widget {
 	}
 
 	public async selectOptionByText(text: string): Promise<void> {
-		await (await this.elem.$(`a*=${text}`)).click()
+		await (await this.byCSS(`a*=${text}`)).click()
```

Comment:
Same as before, consider the use of the proper method from the widget base class.